lib/client.js

```diff
@@ -69,11 +69,16 @@ Client.prototype.sendSMS = function(from, to, message, whendelivered, callback)
   if (!from) throw new Error('The param \'from\' was not supplied.');
   if (!to) throw new Error('The param \'to\' was not supplied.');
   if (!message) throw new Error('The param \'message\' was not supplied.');
-  if (from.length > 11) throw new Error('\'from\' can\'t be longer than 11 '+
-                                        'charcters.');
-  if (!from.match(/^[a-zA-Z0-9]+$/)) throw new Error('\'from\' can only '+
-                                                     'contain a-z, A-Z and '+
-                                                     '0-9');
+  
+  // If "from" is not a phonenumber, do alphanumeric validation
```

Comment:
Could this comment be replaced with `alphanumeric = !from.match(<REGEX>)` and the conditional with `if (alphanumeric)`?

This way, there's no risk of updating the code without updating the comment.